webapp/src/components/chat/chat-list/ChatListItem.tsx

```diff
@@ -55,12 +55,9 @@ const useClasses = makeStyles({
         justifyContent: 'space-between',
     },
     title: {
-        ...shorthands.overflow('hidden'),
-        textOverflow: 'ellipsis',
-        whiteSpace: 'nowrap',
+        ...SharedStyles.overflowEllipsis,
         fontSize: tokens.fontSizeBase300,
         color: tokens.colorNeutralForeground1,
-        lineHeight: tokens.lineHeightBase200,
```

Comment:
removed this to fix clipping issue:
<img width="222" alt="image" src="https://github.com/microsoft/chat-copilot/assets/52973358/d11da410-1c65-4101-99e5-24e4f426cf09">